servers/src/main/java/tachyon/worker/block/allocator/AllocatorType.java

```diff
@@ -33,4 +33,8 @@
    * Allocate to the first storage dir with space
    */
   GREEDY,
+  /**
+   * Round robin allocation to the storage dir
```

Comment:
To be a little bit more clear, also describe if the round robin strategy is across tiers or within the same tier.